src/config/index.js

```diff
@@ -176,12 +176,12 @@ export const navigationArgs = {
   },
   socialLinks: [
     {
-      href: "https://twitter.com/",
+      href: "https://twitter.com/Code4Africa",
       src: Twitter,
       label: "twitter",
     },
     {
-      href: "https://web.facebook.com/?_rdc=1&_rdr",
+      href: "https://www.facebook.com/CodeForAfrica",
```

Comment:
QQ @nerdlet , are we still using these in prod? i.e. aren't we pulling them from CMS?

Reply:
@kilemensi Nope , we shouldn't. Let me remove them.